src/Pos/Wallet/Web/Server/Methods.hs

```diff
@@ -165,8 +169,12 @@ launchNotifier nat =
         , updateNotifier
         ]
   where
-    cooldownPeriod = 5000000         -- 5 sec
+    cooldownPeriod :: Microsecond
+    cooldownPeriod = 5000000
```

Comment:
This comment eliminates necessity to count zeroes if someone wants to understand how big this value is.